data/src/main/java/com/teamagam/gimelgimel/data/location/adpater/LocationSampleDataAdapter.java

```diff
@@ -0,0 +1,47 @@
+package com.teamagam.gimelgimel.data.location.adpater;
```

Comment:
This comment refers to all of the data adapters / mappers.

I think we should create a UniversalDataAdapter, it'll contain all of our adapters (List with interface or explicit members).
The adapter will get an object by method, and will decide which adapter should handle it,